### DISCUSSION TITLE
Error on lib.rs

### DISCUSSION THREAD

**Discussion by Slyterman** [juhaku/utoipa#128] [General]
When I try and build and include utoipa as a dependency either version 0.2.0 or the new 1.0.2 I get this error
`
error[E0599]: no method named `value` found for struct `LitBool` in the current scope
    --> /Users/mikeslyter/.cargo/registry/src/github.com-1ecc6299db9ec823/utoipa-gen-1.0.2/src/lib.rs:1222:42
     |
1222 |             Ok(input.parse::<LitBool>()?.value())
     |                                          ^^^^^-- help: remove the arguments
     |                                          |
     |                                          field, not a method

`
Is this a know issue? I am using rust 1.60.0

**Comment by juhaku** [comment]
Seems odd. Have you tried to run any of those examples in the repo? They should run. Do you have `syn` as your dependency? If yes what version? If you are using utoipa only create and show OpenAPI doc then I guess there is no need to have the `syn` as dependency.

The `LitBool` is type in `syn` crate and if utoipa would not build there is no way to get it to released to the crates IMO. 

I have Rust 1.60 and I can run any of those examples in the utoipa repo without issues. Also witch operating system you are using? Although it should work with any of them. Also if you have just changed or upgraded some dependencies versions you might try to run `cargo clean` to clean the cached versions.

**Reply by Slyterman** [reply]
I am using Mac OS. I updated utoipa-swagger-ui to 1.0.0 and utoipa to 1.0.2. I removed the Cargo.lock file and everything builds now.

**Reply by Slyterman** [reply]
Thanks for your help. I am new to Rust.

**Reply by juhaku** [reply]
Good to hear, no worries.
